localplanning.services/src/components/applications/ApplicationsAuth.tsx

```diff
@@ -0,0 +1,14 @@
+import { ApplicationsList } from "./ApplicationsList";
+import { useSearchParams } from "./hooks/useSearchParams";
+import LoginForm from "./LoginForm";
+
+export const ApplicationsAuth: React.FC = () => {
+  const { token, email } = useSearchParams();
+  const hasUsedMagicLink = Boolean(token && email);
```

Comment:
I've moved this `hasUsedMagicLink` check up a level as handling the permutations of this combined with the various error and success states within `ApplicationsList` was getting quite complex 🍝